test/scholar/covariance/ledoit_wolf.exs

```diff
@@ -0,0 +1,126 @@
+defmodule Scholar.Covariance.LedoitWolfTest do
```

Comment:
I'm not sure if this test is actually running. It's missing the `_test` suffix in the filename